docs-ui/components/form.stories.js

```diff
@@ -212,6 +213,64 @@ storiesOf('Forms|Fields', module)
       </Panel>
     ))
   )
+  .add(
+    'TextareaField',
+    withInfo({
+      text: 'Textarea input',
+      propTablesExclude: [Form],
+    })(() => (
+      <Panel>
+        <Form initialData={{context: {location: 'cat'}}}>
+          <TextareaField
+            name="simpletextfieldvalue"
+            label="Simple Textarea Field with Value"
+            placeholder="Simple Textarea Field"
+            defaultValue="With a value present"
+          />
+          <TextareaField
+            name="simpletextfieldautosize"
+            autosize={boolean('autosize', true)}
+            label="Textarea field with autosize"
+            rows={number('Number of rows', 2)}
+            placeholder="Use knobs to control rows and autosize setting"
+          />
+          <TextareaField
+            name="simpletextfieldvaluedisabled"
+            label="Disabled - Simple Textarea Field with Value"
+            placeholder="Simple Textarea Field"
+            defaultValue="With a value present"
+            disabled
+          />
+          <TextareaField
+            name="simpletextfieldplaceholderdisabled"
+            label="Disabled - Simple Textarea Field with Placeholder"
+            placeholder="This is placeholder text in a disabled field"
+            disabled
+          />
+          <TextareaField
+            name="textfieldwithreturnsubmit"
+            label="Textarea Field With Return Submit"
+            placeholder="Type here to show the return button"
+            showReturnButton
+          />
+          <TextareaField
+            name="textfieldflexiblecontrol"
+            label="Textarea Field With Flexible Control State Size"
+            placeholder="Type text and then delete it"
+            required
+            flexibleControlStateSize
+          />
+          <TextareaField
+            name="textfielddisabled"
+            label="Textarea Field with disabled reason"
+            placeholder="I am disabled"
+            disabled
+            disabledReason="This is the reason this field is disabled"
+          />
+        </Form>
```

Comment:
`FormField` has `help` and `error` messages. It would be a good idea to show how `TextareaField` would look with these additional states.